lib/buildFactory.js

```diff
@@ -388,11 +388,20 @@ class BuildFactory extends BaseFactory {
      * @param  {Object}   config.paginate         Pagination parameters
      * @param  {Number}   config.paginate.count   Number of items per page
      * @param  {Number}   config.paginate.page    Specific page of the set to return
+     * @param  {Boolean}  [config.fetchSteps]     Fetch steps for all the builds, default is true
      * @return {Promise}                          Resolve builds after merging with step models
      */
     list(config) {
+        const fetchSteps = config.fetchSteps ? config.fetchSteps : false;
```

Comment:
does this change has to go the very last ? Alternatively I would just default this to `true` so that no-one breaks.

Reply:
that's what i think in the first place to default it to `true` and that requires a lot of changes in our api and models to pass in the flag. Went through the code with Dao and we're not using / modifying steps when list builds. So it should be safe. Make it a flag so that later on if we need it we can always pass it